akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -271,15 +271,20 @@ By default, these remoting features are disabled when not using Akka Cluster:
 * Remote Deployment: falls back to creating a local actor
 * Remote Watch: ignores the watch and unwatch request, and `Terminated` will not be delivered when the remote actor is stopped or if a remote node crashes
 
-When used with Cluster, all previous behavior is the same except a remote watch of an actor is no longer possible before a node joins a cluster, only after.
+When used within a Cluster, all previous behavior is the same except a remote watch of an actor is no longer possible before a node joins a cluster, only after.
```

Comment:
Here's a bit of a suggestion for a restructuring of that paragraph:

```
Watching an actor on a node outside the cluster may have unexpected
@ref[consequences](../remoting-artery.md#quarantine), such as quarantining
so it has been disabled by default in Akka 2.6. This is either the case if
cluster is not used at all (but only plain remoting) or when watching an actor outside of the cluster.

On the other hand, failure detection between nodes, that are members of the same cluster,
do not have that shortcoming. Thus, when remote watching or deployment is used within
a cluster, they are working the same in 2.6 as before except that a remote watch of
an actor will be ignored (?correct?) before a node has joined a cluster and works only after.

To optionally enable...
```

Reply:
Great, updated with slight rewording.